classes/class-gf-securesubmit.php

```diff
@@ -20,6 +20,11 @@ class GFSecureSubmit extends GFPaymentAddOn
     /**
      * @var bool
      */
+    private $isCert = false;
```

Comment:
Is this used anywhere?

Reply:
Nope. But, I can see if I can use it to set the CCA Js endpoint

Reply:
OK. Got this worked out.